### DISCUSSION TITLE
Access AWS EKS clusters via mitmproxy

### DISCUSSION THREAD

**Discussion by sazecis** [mitmproxy/mitmproxy#5530] [Q&A]
I need to log all **kubectl** commands (including exec) going through a bastion host. For the logging I planned to use the mitmproxy.
My cluster uses [aws-iam-authenticator](https://github.com/kubernetes-sigs/aws-iam-authenticator) for user/role authentication.

I am struggling to bring together a solution where my mitmproxy and it's certificates get trusted.

The base setup which I tried is simply starting the mitmproxy on an arbitrary port:
 - mitmdump -p 9090
After this I try to call a simple kubectl command:
- https_proxy=:9090 kubectl get ns

But I get the following errors:
Kubectl: Unable to connect to the server: x509: certificate signed by unknown authority
mitmdump: Certificate verification error for <...>.gr7.eu-central-1.eks.amazonaws.com: unable to get local issuer certificate (errno: 20, depth: 0

Do you have ideas how could I solve this issue? How I see mitmproxy cannot interact with aws credentials and that is causing the problem.

**Comment by suresh-tricog** [comment]
Facing same issue, any luck ?
